### PR TITLE
[T-10] travis&husky

### PR DESCRIPTION
**Pull request by ShevchenkoPavel**

### CODE REVIEW

**Comment by ShevchenkoPavel** [comment]
#2

File: src/serviceWorker.ts [hunk 1]
```diff
@@ -24,54 +30,16 @@ type Config = {
...
-      } else {
-        // Is not localhost. Just register service worker
-        registerValidSW(swUrl, config);
-      }
-    });
-  }
-}
-
-function registerValidSW(swUrl: string, config?: Config) {
+function registerValidSW(swUrl: string, config?: Config): void {
```
Comment (oTkPoBeHuE) [inline src/serviceWorker.ts:34]: add padding on top

File: src/serviceWorker.ts [hunk 2]
```diff
@@ -105,8 +73,7 @@ function registerValidSW(swUrl: string, config?: Config) {
       console.error('Error during service worker registration:', error);
     });
 }
-
-function checkValidServiceWorker(swUrl: string, config?: Config) {
+function checkValidServiceWorker(swUrl: string, config?: Config): void {
```
Comment (oTkPoBeHuE) [inline src/serviceWorker.ts:78]: add padding on top

File: src/serviceWorker.ts (hunk 1 above)
Comment (ShevchenkoPavel) [reply src/serviceWorker.ts:34]: fixed

File: src/serviceWorker.ts (hunk 2 above)
Comment (ShevchenkoPavel) [reply src/serviceWorker.ts:78]: fixed

**Pull request merged**